src/style/CharityPage.less

```diff
@@ -76,6 +76,16 @@
 			margin-bottom: 30px;
 			.charity-logo {
 				height: 60px;
+			.charity-name {
+				text-transform: capitalize;
+				font-family: 'Nunito Sans';
+				font-weight: 800;
+				font-size: 32px;
+				color: #414141;
```

Comment:
To keep the css consistent and easy to maintain, we normally want to define a few font and colour settings in one file, then reference those.

NB: I can't recall how well organised the SoGive css is -- there's probably an existing file for that, and it's probably only partly used.

Reply:
NB: When you do want local font settings -- just leave a comment as to why they the widget doesn't follow the general website fonts.